src/core/thread/thread_netif.cpp

```diff
@@ -171,6 +172,28 @@ ThreadError ThreadNetif::RouteLookup(const Ip6::Address &source, const Ip6::Addr
     return error;
 }
 
+ThreadError ThreadNetif::TmfFilter(const otMessage *aMessage, const otMessageInfo *aMessageInfo)
```

Comment:
Do you think that we should expose this interceptor feature to user for the application CoAP server?